js/src/personal/kelly/Kelly.page.tsx

```diff
@@ -5,30 +5,128 @@ import { theme } from './theme.ts'
 export function KellyPage() {
   return (
     <MantineProvider theme={theme}>
-      <div className={styles.body_load}></div>
-      <div className={styles.nav}>
-        <a>
-          <p className={styles.nav_item}>{'About'}</p>
-        </a>
-        <a>
-          <p className={styles.nav_item}>{'Projects'}</p>
-        </a>
-        <a>
-          <p className={styles.nav_item}>{'Contact'}</p>
-        </a>
-      </div>
-      <div className={styles.body_container}>
-        <Title className={styles.title}>{'KELLY LIN'}</Title>
-        <div className={styles.window_container}>
-          <Box className={styles.window}></Box>
+      <div className={styles.body_wrap}>
+        <link
+          rel="preconnect"
+          href="https://fonts.googleapis.com"
+        />
+        <link
+          rel="preconnect"
+          href="https://fonts.gstatic.com"
+        />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Libre+Caslon+Text:ital,wght@0,400;0,700;1,400&display=swap"
+          rel="stylesheet"
+        />
+        <div className={styles.nav}>
```

Comment:
Should likely break this out as it's getting quite long. Might make sense to have a Navigation.tsx and import it as a component here, but it does make handling the refs harder if you're using useScrollIntoView

Reply:
I will begin separating the sections more :))